src/Symfony/Component/Intl/Resources/emoji/Makefile

```diff
@@ -5,7 +5,7 @@ update: ## Update sources
 	@composer update
 	@curl https://api.github.com/emojis > vendor/github-emojis.json
 	@curl https://raw.githubusercontent.com/iamcal/emoji-data/master/emoji.json > vendor/slack-emojis.json
-	@curl https://unicode.org/Public/emoji/latest/emoji-test.txt > vendor/emoji-test.txt
+	@curl -L https://unicode.org/Public/emoji/latest/emoji-test.txt > vendor/emoji-test.txt
```

Comment:
The previous URL was a 301 and thus we save some HTML. Now curl follow the redirection